auth/src/main/java/org/aerogear/mobile/security/SecurityService.java

```diff
@@ -0,0 +1,55 @@
+package org.aerogear.mobile.security;
+
+import org.aerogear.mobile.core.MobileCore;
+import org.aerogear.mobile.core.ServiceModule;
+import org.aerogear.mobile.core.configuration.ServiceConfiguration;
+import org.aerogear.mobile.core.metrics.MetricsService;
+import org.aerogear.mobile.security.impl.SecurityCheckExecutorImpl;
+import org.aerogear.mobile.security.metrics.SecurityCheckResultMetric;
+
+public class SecurityService implements ServiceModule{
```

Comment:
Service vs. Module ?

Reply:
With Auth and Metrics, the two most recent additions we went with Service. So I think it's service now

Reply:
a service is not a module :)